docs/CONTRIBUTING.md

```diff
@@ -51,13 +51,42 @@ the mailing list have a tendency to fall through the cracks.
     entries to `src/types.db`, you should talk to us early in the design
     process.
 
+### Labels
+
+PRs need to be categorized into one of three categories:
+
+*   **Feature**
+
+    The code has new behavior. Performance and documentation improvements are
+    considered features. These changes would typically go into a "feature
+    release".
+*   **Bug**
+
+    The code fixes a bug, regression, or otherwise unintended behavior. These
+    changes would typically go into a "patch" release.
+*   **Unlisted Change**
```

Comment:
I'd like more descriptive name for this category, maybe "User invisible change"?

Reply:
"Not user visible" would be marginally shorter.

Reply:
The name was initially chosen to say "not listed in the change log". What do you think about "Internal"?

Reply:
I have seen "Maintenance" on a different project and think this is probably the best term overall. Changed.

Reply:
Yes, "Maintenance" is definitely best!